lib/net/lambda.dart

```diff
@@ -0,0 +1,39 @@
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+import 'package:amplify_flutter/amplify.dart';
+import 'package:aws_lambda_api/lambda-2015-03-31.dart';
+
+class LambdaWrapper {
```

Comment:
This should not be the recommended way to invoke lambda from amplify. Instead, we do it like this, https://docs.amplify.aws/cli-legacy/graphql-transformer/function/#function

You can refer to `/Users/zhangqi/ws/toktik/amplify/backend/function/listPostExs/src/index.js` . 

Feel free to sync up with me about how to make it work.